front/src/app/core/home-page/home-page.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+import {IsLogoutHomePageComponent} from './is-logout-home-page/is-logout-home-page-component';
+import {IsLoginHomePageComponent} from './is-login-home-page/is-login-home-page.component';
+import {AuthService} from '../login/services/auth.service';
+
+@Component({
+  selector: 'app-home-page',
+  standalone: true,
+  imports: [IsLoginHomePageComponent, IsLogoutHomePageComponent],
+  templateUrl: './home-page.component.html',
+  styleUrl: './home-page.component.scss'
+})
+export class HomePageComponent {
+  isLogin: boolean = false;
+  userName: string | null = null;
+
+  constructor(private authService: AuthService) {}
+
+  ngOnInit() {
+    this.authService.user$.subscribe((user) => {
+      this.isLogin = !!user;
```

Comment:
todo: la syntaxe `!!x` c'est un hack JS c'est pas une bonne habitude, en plus c'est pas super lisible

récupère ce fichier de code (+ les test juste à coté) : https://github.com/kuroidoruido/js-libs/blob/main/libs/fp/src/predicates.ts

```suggestion
      this.isLogin = isDefined(user);
```